site/pages/react/how-to-set-up-smart-account-client.mdx

```diff
@@ -0,0 +1,53 @@
+| `type`              |                         `SupportedAccountTypes` |                                                                                                                                                                                        options are "ModularAccountV2", "LightAccount", "MultiOwnerLightAccount", "MultiOwnerModularAccount" |
+| `policyId`          |                                        `string` |                                                                                                                                                                                                       gas manager policy ID from the [Dashboard](https://dashboard.alchemy.com/gas-manager) |
+| `useSimulation`     |                                       `boolean` |                                                                                                                                                                      enables use of [`alchemyUserOperationSimulator`](/reference/account-kit/infra/functions/alchemyUserOperationSimulator) |
+| `customMiddleware`  |                            `ClientMiddlewareFn` |                                                                                                                                                                                                                                      custom middleware to run before requesting sponsorship |
+| `feeEstimator`      |                            `ClientMiddlewareFn` |                                                                                 function that estimates the fees for a user operation (if you are using our RPCs, it's important to use the [`alchemyFeeEstimator`](/reference/account-kit/infra/functions/alchemyFeeEstimator) middleware) |
+| `gasEstimator`      |
+| `signUserOperation` |                            `ClientMiddlewareFn` |                                                                                                                                                                                                                            function that sends a user operation or catch of user operations |
+| `opts`              | ` z.input<typeof SmartAccountClientOptsSchema>` |                                                                                                                                                                                                                                                                             further options |
+| `accountParams`     |                                 `AccountConfig` | object with the following fields <ul><li>`mode` : "default" \| "7702"</li><li>`entryPoint`: of type `EntryPointDef`</li><li>`signerEntity`: of type `SignerEntity`</li><li>`salt`: of type `bigint`</li><li>`factoryAddress`: of type `Address`</li><li>`initCode`: of type `Hex`</li></ul> |
```

Comment:
certain params only work w/ certain account types, i.e. `mode` is only available for `ModularAccountV2`.
not sure if there is an easy way to denote that in the docs or not though.